tests/test_auth.py

```diff
@@ -20,13 +20,3 @@ def test_testing_login_fail(self):
         app.config['TESTING_LOGIN'] = False
         response = app.test_client().get('/testing-login/')
         self.assert_404(response)
-
-    def test_restricted(self):
```

Comment:
I think so. It just needs to incorporate some elements of frontend testing - checking if certain elements exist on the page.